configs/common/packages.yaml

```diff
@@ -233,13 +232,15 @@ packages:
     require: '~mpi'
   # https://github.com/JCSDA/spack-stack/issues/1276
   py-matplotlib:
-     require: '@3.7.4'
+    require: '@3.7.4'
   # Variant ~mpi not working for latest py-netcdf4
   # https://github.com/spack/spack/issues/47652
   py-netcdf4:
     require: '@1.7.1 +mpi'
-  py-pandas:
-    require: '+excel'
+  # To avoid duplicate packages
```

Comment:
Wouldn't the concretizer `unify: true` prevent duplication?

Reply:
It will lead to concretization errors if `unify: true` is set. Unfortunately, the concretizer isn't perfect, it still has its bugs. But I am happy to move this (numpy and cython) into the cylc-dev template instead of applying it to all environments for now (but new packages that are coming down the road will want the same settings  for unified-env/neptune-env, too).